### PR TITLE
Add Highrise Blockchain Testnet info

### PR DESCRIPTION
**Pull request by mculinovic**

### CODE REVIEW

File: networks/README.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ WARNING: this information is provided by the community. Please exercise caution
...
-| 432201   |    Fuji |        Dexalot Fuji Subnet |
-| 335      |    Fuji |  DeFi Kingdoms Fuji Subnet |
-| 808      |    Fuji | Portal Fantasy Fuji Subnet |
+| Chain ID | Network |                        Name |
+| :------- | ------: | --------------------------: |
+| 11111    |    Fuji |                       WAGMI |
+| 432201   |    Fuji |         Dexalot Fuji Subnet |
+| 335      |    Fuji |   DeFi Kingdoms Fuji Subnet |
+| 808      |    Fuji |  Portal Fantasy Fuji Subnet |
+| 8184     |    Fuji | Highrise Blockchain Testnet |
```
Comment (aaronbuchwald) [inline networks/README.md:20]: Have you made a PR to add 8184 to chainlist?

File: networks/testnet/8184/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Social links
+
+|      Type |                                   URL |
+| --------: | ------------------------------------: |
+| Instagram |    https://www.instagram.com/highrise |
+|   Twitter |       https://twitter.com/HighriseApp |
+|  Facebook | https://www.facebook.com/highrisegame |
+|   Tik Tok |  https://www.tiktok.com/@joinhighrise |
+|   Discord |        https://discord.com/invite/hcc |
+|    Medium |       https://medium.com/pocketworlds |
```
[the remark this answers is not in the document]
Comment (mculinovic) [reply networks/testnet/8184/README.md]: Done

File: networks/README.md (hunk 1 above)
Comment (mculinovic) [reply networks/README.md:21]: Here is the PR https://github.com/ethereum-lists/chains/pull/1579 
Pending approval

**Comment by aaronbuchwald** [comment]
Closing this PR in favor of migrating to the public-chain-assets repo: https://github.com/ava-labs/public-chain-assets

**Pull request closed without merging**